src/blocks/CallToAction/index.tsx

```diff
@@ -24,32 +26,46 @@ export const CallToAction: React.FC<CallToActionProps> = props => {
   return (
     <BlockSpacing>
       <Gutter className={classes.callToAction}>
-        <div data-theme="dark">
-          <div className={classes.bgWrapper}>
-            <Gutter disableMobile className={classes.bgGutter}>
-              <div className={classes.bg1}>
-                {/* <div className={classes.pixelBG}>
-              <PixelBackground />
-            </div> */}
-              </div>
-            </Gutter>
-          </div>
-          <div className={[classes.contentWrap, 'grid'].filter(Boolean).join(' ')}>
-            <div className={'cols-8 cols-m-8'}>
+        <div className={[classes.wrapper].filter(Boolean).join(' ')}>
+          <BackgroundGrid ignoreGutter />
+          <div className={[classes.container, 'grid'].filter(Boolean).join(' ')}>
+            <div className={[classes.contentWrapper, 'cols-7 cols-m-8'].filter(Boolean).join(' ')}>
               <RichText content={richText} className={classes.content} />
             </div>
-            <div className={'cols-7 start-11 start-l-10 cols-m-8 start-m-1'}>
+            <div
+              className={[classes.linksContainer, 'cols-8 start-9 cols-m-8 start-m-1 grid']
+                .filter(Boolean)
+                .join(' ')}
+            >
+              <BackgroundScanline
+                className={[classes.scanline, 'cols-16 start-4 start-l-3 cols-m-8 start-m-1']
+                  .filter(Boolean)
+                  .join(' ')}
+                crosshairs={['top-left', 'bottom-left']}
+              />
+              {/* Double printing the BackgroundGrid component here so that it displays above the scanline */}
+              <BackgroundGrid ignoreGutter />
+              <CrosshairIcon className={[classes.crosshairTopLeft].filter(Boolean).join(' ')} />
+              <CrosshairIcon className={[classes.crosshairBottomRight].filter(Boolean).join(' ')} />
               {feature === 'cpa' && (
```

Comment:
@paulpopus Is this `feature` prop / `CreatePayloadApp` component staying in the CTA? It looks pretty janky when it's rendered right now.

![Screen Shot 2024-01-30 at 10 42 08 AM](https://github.com/payloadcms/website/assets/35232443/b70a2f19-3902-4fda-9c87-9eac1d9eb091)

Reply:
Removed